spidermon/utils/web.py

```diff
@@ -0,0 +1,20 @@
+import sys
+import logging
+from six.moves.urllib.parse import urlparse
```

Comment:
The functions of this module are so small, and used only in spidermon/contrib/validation/jsonschema/tools.py that I think they would be better placed in the same module. This would improve readability, as we won't need to jump between modules to see functions simple like that.